FRPs/proposal-antitrust.md

```diff
@@ -0,0 +1,44 @@
+Increasing Competition Using Decentralized Technologies
+
+(1) Overview
+
+The Decentralization Research Center (DRC) is excited to submit the research proposal below. We are a nonprofit organization that supports nonpartisan decentralization research and acts as a hub for critical stakeholders. Our mission is to encourage the equitable ownership, governance, and regulation of emerging technologies that impact our everyday lives.
+
+In our proposed research, we aim to investigate how decentralization, especially as enabled by blockchain technology, can impact competition and offer solutions to monopolistic tendencies in various sectors. Central to this investigation is the role of MEV in fostering competitive environments and their implications for future antitrust policies and regulations. The study is inspired by some of our previous research, the paper we co-authored for the Harvard Kennedy School on equitable ownership and governance, and discussions with MEV builders.
+
+The hypothesis suggests that decentralized structures and technologies, including those underpinned by blockchain, can disrupt traditional power concentrations and foster a more equitable distribution of ownership and governance. The methodology involves a three-phase research process over six months. The first two months will focus on literature review and case study selection, including data collection through interviews and market research. The subsequent two months will be dedicated to writing and peer-reviewing the research paper. In the final phase, the team will draft shorter, accessible research outputs like blog posts and tweet threads, culminating in a presentation at the next DRC summit, which will likely be in DC in June 2024.
```

Comment:
This is really great, clear outline. For the FRP itself, we typically conduct FRPs over 2 months for the FRP grant. We can either try to collapse phase 1 & 2 together or add a deliverable after phase 1 (the first two months) that demonstrates work accomplished (a summary of literature review and case study overview) and kicks off a second FRP for the research paper itself. We can discuss this more!

Reply:
Let's do it. Would you mind emailing me and we can set it up and also thereafter a slack channel for comms moving forward? 

reid@flashbots.net